internal/pkg/config/output.go

```diff
@@ -56,8 +63,15 @@ func (c *Elasticsearch) InitDefaults() {
 
 // Validate ensures that the configuration is valid.
 func (c *Elasticsearch) Validate() error {
-	if c.APIKey != "" {
-		return fmt.Errorf("cannot connect to elasticsearch with api_key; must use service_token")
+	if c.ServiceToken == "" {
```

Comment:
Our validation will now ensure that there is some service_token value; either directly specified, or from a non-empty file.

Reply:
nitpick :does it make sense to create a function here for the serviceToken validation it start to be a few if imbrication

Reply:
I don't think this complex enough